### PR TITLE
Menu Entry Swapper: Added Fairy Tree (poh) functionality

### PR DESCRIPTION
**Pull request by sunha7**
Just got a fairy tree in my poh and realised the swaps didn't work

### CODE REVIEW

**Comment by sunha7** [comment]
borked it let me try again

**Comment by sunha7** [comment]
ok im somewhat new to using git and frankly cant figure out why there's a conflict. If you know, do share please.

This is tested & working fine in my own build without any issues. If someone could look at this for approval that'd be great.

**Comment by xKylee** [comment]
branch is waaaaay out of date, cant even fix conflicts :(

**Comment by xKylee** [comment]
superseded by 1555
